src/shared/auth/wsClient.ts

```diff
@@ -39,7 +39,7 @@ export const createAuthServerWsClient = (authServerUrl?: string) => {
 
   const sendSuccessfulOutcome = async (requestId: string, sender: string, result: any): Promise<void> => {
     try {
-      await request<OutcomeResponse>('outcome', {
+      await request<void>('outcome', {
```

Comment:
Although it's not being used, it should return an `OutcomResponse`, right? If that's the case, we might want to revert this change.